jujugui/static/gui/src/app/components/usso-login-link/_usso-login-link.scss

```diff
@@ -0,0 +1,26 @@
+.usso-login {
+  position: relative;
+
+  &__notification {
+    background: #fff;
+    box-shadow: 0 1px 4.85px 0.15px rgba(0,0,0,0.2);
```

Comment:
To pass a linter. Can you remove the zeros before decimals.

Also there should be a space after commas.

Therefore:  `box-shadow: 0 1px 4.85px .15px rgba(0, 0, 0, .2);`